src/rust/ide/lib/parser/src/wsclient.rs

```diff
@@ -211,7 +241,16 @@ impl Client {
         let response = self.rpc_call(request)?;
         match response {
             Response::Success {module} => Ok(module),
-            Response::Error {message}  => Err(ParsingError(message)),
+            Response::Error  {message} => Err(ParsingError(message)),
+        }
+    }
+
+    pub fn generate_html_docs(&mut self, program:String) -> api::Result<String> {
```

Comment:
Please add all pragmas that we have in other libraries to this library. Especially pragmas about warnings when no docs are provided. Linter should catch such places.

Reply:
Linter probably doesn't catch it because of this pragma at the beginning of the module : `#![cfg(not(target_arch = "wasm32"))]`, all pragmas in lib.rs are there. And it is necessary because of usage of 2 connection methods : jsclient for scala-parser in JS, and wsclient, for connection with parser-service

Reply:
If so, then just fix the code pls :)